### DISCUSSION TITLE
Reference terms of the transaction level contract

### DISCUSSION THREAD

**Discussion by jainneerajnj** [ONDC-Official/protocol-network-extension#79] [Announcements]
The [Reference Terms](https://docs.google.com/document/d/1hKImxDwZ-LUd1ln0D61KhhuQ9-Rqrzt2/edit#) are terms which will not vary from one contract to another with respect to transactions on the ONDC Network. These Reference Terms along with the commercial terms agreed to by the Buyer App and Seller App (or Logistics Buyer and Logistics Seller) through the protocol (such as buyer finder fee, settlement window etc.) will constitute the entire agreement in relation to one transaction on the ONDC Network.

The Reference Terms will be accessible on a static URL hosted by ONDC. As the Network expands and grows to allow a greater variety of products and services, the Reference Terms may be revised, or ONDC may create multiple versions of Reference Terms.
The Reference Terms are incorporated into the digital contract by reference using the static URL.

**Comment by adithya-wlfd** [comment]
@jainneerajnj The link to the reference terms isn't publicly accessible. Can you check if the link needs to be updated?
